src/ray/util/event.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ray/util/event.h"
+
+namespace ray {
+
+EventManager &EventManager::Instance() {
+  static EventManager INSTANCE;
```

Comment:
Please just use lowercases! I think that's the convention (based on other parts of code. )